primus.js

```diff
@@ -621,15 +621,29 @@ Primus.prototype.initialise = function initialise(options) {
     }
   }
 
+  primus.addGlobalListeners();
+
+  return primus;
+};
+
+Primus.prototype.addGlobalListeners = function () {
```

Comment:
Well the initialise method already contains all the logic that should only be executed once when a new instance is created. Adding another method that should only be called once just doesn't make a lot of sense to me. I do understand that having `addGlobalListeners` makes sense if you also have a `removeGlobalListeners` but the latter method isn't strictly required either as the `destroy` function can also just run the `after` hook as a function. (see https://github.com/unshiftio/demolish for docs)